### DISCUSSION TITLE
Returning one or more messages to the UI upon execution of a manually triggered Flow

### DISCUSSION THREAD

**Discussion by colemahatmccreary** [directus/directus#14201] [Feature Requests]
Similar to the Log to Console operation, this would enable a separate message/series of messages to be returned to the App user in a modal upon the flow's execution.

The default "Flow XXX ran successfully" message can be ambiguous in the case of Flows with various conditions, etc - this would enable different messages for each branch of the flow's logic.

Maybe just a WYSIWYG editor like the email/notification interface?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
